public/docs/ts/latest/guide/index.jade

```diff
@@ -30,85 +22,68 @@ table(width="100%")
     td <b><a href="../quickstart.html">QuickStart</a></b>
     td
       :marked
-        The foundation for every page and sample in this documentation.
+        A first taste of Angular with zero installation. 
+        Run "Hello World" in an online code editor and start playing with live code.
   tr(style=top)
-    td <b><a href="./">Guide</a></b>
+    td <b>Guide</b>
     td
       :marked
-        The essential ingredients of Angular development.
+        Learn the Angular basics (you're already here!).
   tr(style=top)
-    td <b><a href="../api">API Reference</a></b>
+    td <b><a href="../api/">API Reference</a></b>
     td
       :marked
         Authoritative details about each member of the Angular libraries.
   tr(style=top)
-    td <b><a href="../tutorial">Tutorial</a></b>
+    td <b><a href="../tutorial/">Tutorial</a></b>
     td
       :marked
         A step-by-step, immersive approach to learning Angular that
         introduces the major features of Angular in an application context.
   tr(style=top)
-    td <b><a href="!{advancedUrl}">Advanced</a></b>
+    td <b><a href="ngmodule.html">Advanced</a></b>  
```

Comment:
Please revert the href as `!{advancedUrl}` (and drop the trailing space on the line).